protocol-configuration/permissions.ts

```diff
@@ -29,9 +29,14 @@ export const permissions = {
   BALANCER_MANAGER_ADMIN_ROLE: [],
   PSM_ADMIN_ROLE: [],
   TRIBAL_CHIEF_ADMIN_ROLE: ['optimisticTimelock', 'tribalChiefSyncV2'],
```

Comment:
TODO: This `permissions.ts` will become out of date when the `TribalCouncil` starts creating and granting roles